### PR TITLE
[45] Product Page Using HTML

### PR DESCRIPTION
**Pull request by ayotola-t**
## Task: `Product Page for a e-commerce store using HTML only`
Screenshot 2020-09-24 at 6 58 11 PM
Screenshot 2020-09-24 at 6 58 24 PM

### CODE REVIEW

**Review by adaugochi — CHANGES_REQUESTED** [review]
I think you are required to just use two images for this task

Also, you are meant to show the view on your browser, not your git commands

**Comment by ayotola-t** [comment]
> I think you are required to just use two images for this task
> 
> Also, you are meant to show the view on your browser, not your git commands

There was no specification as to the number of images in the task. I'm adding a screenshot of the instructions. 
IMG_7306

Here are screenshots of the browser view:

Screenshot 2020-09-24 at 8 41 21 PM
Screenshot 2020-09-24 at 8 41 34 PM

**Comment by ayotola-t** [comment]
Screenshot 2020-09-24 at 8 41 57 PM

**Review by adaugochi — COMMENTED** [review]
Well done

But, you are expected to use just two images for this task, Do ensure to follow the instructions given

**Comment by adaugochi** [comment]
This information should guide you

> Say for instance a Mentor comment of your PR, how do I go about it? How will the update be reflected on the Pull Request I already created? Below is a step to follow:

- Still, on that same branch, update your changes.

- Add your changes (git add .).

- Commit your changes (git commit  -m  "commit message").

- Finally, Push your change (git push origin branch_name)

**Comment by ayotola-t** [comment]
> Well done
> 
> But, you are expected to use just two images for this task, Do ensure to follow the instructions given

Hello, 

I have updated to two images. 

But I'm honestly still a bit confused. Please where were we asked to use just two images for the task (I attached a screenshot previously)? I have re-read again and again.  I feel like there is something I'm missing or I'm not interpreting tasks correctly.

File: submissions/ayotolaTehingbola/html/product.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                <ol> 
+                 <li> <h4> <a href="#"> The Ilesha Bag, $62 </a></h4></li>
+                     <img src="img/Bag.png" alt="raffia and leather bag"> 
+                </ol>
+          </div>
+        </section>
+
+    </body>
+
+    <footer>
```
Comment (ToluAkin) [inline submissions/ayotolaTehingbola/html/product.html:55]: The footer should be in the body tag @ayotola-t

**Review by ToluAkin — CHANGES_REQUESTED** [review]
@ayotola-t Apologies for the confusion on the number of images to use, it was suppose to be added to the success criteria but it was omitted. You are meant to use 2 product images.
Feedback on Pull request
- I think your image names should be in lowercase with no spaces in between.
- You should remove the images you are not using.
- Always create a new branch for every new task

**Pull request closed without merging**